Mathlib/Data/PNat/Prime.lean

```diff
@@ -123,6 +123,30 @@ theorem prime_two : (2 : ℕ+).Prime :=
   Nat.prime_two
 #align pnat.prime_two PNat.prime_two
 
+instance fact_prime_two : Fact (2 : ℕ+).Prime :=
+  ⟨prime_two⟩
+
+instance fact_prime_two' : Fact ((2 : ℕ+) : ℕ).Prime :=
+  ⟨prime_two⟩
+
+theorem prime_three : (3 : ℕ+).Prime :=
+  Nat.prime_three
+
+instance fact_prime_three : Fact (3 : ℕ+).Prime :=
+  ⟨prime_three⟩
+
+instance fact_prime_three' : Fact ((3 : ℕ+) : ℕ).Prime :=
+  ⟨prime_three⟩
+
+theorem prime_five : (5 : ℕ+).Prime :=
+  Nat.prime_five
+
+instance fact_prime_five : Fact (5 : ℕ+).Prime :=
+  ⟨prime_five⟩
+
+instance fact_prime_five' : Fact ((5 : ℕ+) : ℕ).Prime :=
```

Comment:
Is it not possible to deduce this with an instance going from `Fact (p : ℕ+).Prime` to `Fact ((5 : ℕ+) : ℕ).Prime`?

Reply:
Something like this:
```
instance {p : ℕ+} [h : Fact p.Prime] : Fact (p : ℕ).Prime := h
```

Reply:
Trying it.